wsgi_kerberos.py

```diff
@@ -66,11 +66,13 @@ class KerberosAuthMiddleware(object):
     '''
     WSGI Middleware providing Kerberos Authentication
 
-    If no hostname is provided, the name returned by socket.gethostname() will
-    be used.
-
     :param app: WSGI Application
-    :param hostname: Hostname for kerberos name canonicalization
+    :param hostname: Optional hostname to check whether a corresponding server
+        principal has been configured at init time (if not, a warning will be
+        logged), and to use when calling authGSSServerInit at authentication
+        time. If not provided, empty string will be passed to authGSSServerInit
+        so that clients can access the service by any name for which a key
+        exists in the keytab.
```

Comment:
A technically correct description, but maybe confusing to people unfamiliar with kerberos internals. How about something simpler like this:
```
Force the server to only accept requests for the specified hostname. If not specified, clients can access the service by any name in the keytab.
```

Reply:
Good call! Done in the latest revision.